tests/test_model.py

```diff
@@ -130,6 +130,7 @@ class UnknownType():
         for op, method, cls in operators:
             with self.subTest(method):
                 self.assertIsInstance(op(x, y), cls)
+                self.assertIsInstance(op(x, 5.7), cls)
```

Comment:
Ah, I see that this is itself a bunch of tests. Still, a few more explicit ones wouldn't hurt I don't think